demand-capacity-mgmt-backend/pom.xml

```diff
@@ -54,7 +54,7 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-webflux</artifactId>
-            <version>3.1.5</version>
+            <version>3.2.5</version>
         </dependency>
 
         <dependency>
```

Comment:
As you are touching this file, can you please remove duplicate dependency in line 82 though you did not add it
```xml
<dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-data-jpa</artifactId>
        </dependency>
```